packages/admin/cms-admin/src/generator/future/generateGrid.ts

```diff
@@ -324,6 +324,21 @@ export function generateGrid(
 
         let gridType: "number" | "boolean" | "dateTime" | "date" | undefined;
 
+        let filterOperators: string | undefined;
+        if (column.type !== "combination" && column.filterOperators) {
```

Comment:
why type != "combination"?

if an explicit filterOperators is specified, shouldn't we be able to use it?